trigger/host.go

```diff
@@ -192,5 +212,17 @@ func (t *hostTriggers) hostExpiration(sub *event.Subscription) (*notification.No
 		}
 	}
 	t.templateData.ExpirationTime = t.host.ExpirationTime.In(timeZone).Format(time.RFC1123)
-	return t.generate(sub)
+	return t.generateExpiration(sub)
+}
+
+func (t *hostTriggers) spawnHostIdle(sub *event.Subscription) (*notification.Notification, error) {
+	shouldNotify, err := t.host.ShouldNotifyStoppedSpawnHostIdle()
+	if err != nil {
+		return nil, err
+	}
+	if !shouldNotify {
+		return nil, nil
+	}
+	t.templateData.LastCommunicationTime = t.host.LastCommunicationTime.Format(time.RFC1123)
```

Comment:
Use the last stop event time here instead of last communication time, which is probably set to when they first started the host.

Reply:
I just hardcoded this to three months in the template message, do you think that's alright haha or i can turn the timestamp i'm searching for into a global and pass it in

Reply:
I think 3 months is an okay assumption to hard-code for now. I doubt we'll be changing this very frequently, and it's not that hard to make it configurable if we really want in the future.